### PR TITLE
15 ausrüstung main page equipment items anzeigen

### PR DESCRIPTION
**Pull request by jstHH**
Anzeige der Ausrüstungsitems hinzugefügt (Backend und Frontend)

### CODE REVIEW

File: frontend/src/components/EquipmentItemCard.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import {EquipmentItem} from "../model/EquipmentItem";
+import {Card} from "react-bootstrap";
+
+
+export type EquipmentItemCardProps = {
+    equipmentItem: EquipmentItem
+}
+
+export default function EquipmentItemCard({equipmentItem}: EquipmentItemCardProps) {
+    return <div>
```
Comment (Gadenko) [inline frontend/src/components/EquipmentItemCard.tsx:10]: If i find nothing would be stupid.

Comment (Gadenko) [inline frontend/src/components/EquipmentItemCard.tsx:10]:
```suggestion
    return 
    <div>
```
If i write nothing is not good :P

File: backend/src/main/java/com/example/backend/dto/EquipmentItemDTO.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class EquipmentItemDTO {
```
Comment (Flooooooooooorian) [inline backend/src/main/java/com/example/backend/dto/EquipmentItemDTO.java:14]: For what is this used for?
If you dont need it, remove it (for now).

**Review by Gadenko — COMMENTED** [review]
Your Code looks good. I wrote just a think what i like. But is maybe not what you like xD. I did it not in everline, like this return<div>.

**Review by Flooooooooooorian — APPROVED** [review]
LGTM 🏖️

**Pull request merged**